src/util/index.js

```diff
@@ -0,0 +1,6 @@
+export * from './zeroInsteadOfNegative';
+export * from './normBy';
+export * from './normProj';
+export * from './getMax';
+export * from './getMedians';
+export * from './dimMADstd';
```

Comment:
This file is useless. You may directly import the right util method that you need